### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-docker.git`](https://github.com/cisagov/skeleton-docker.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/domain-manager-api.git domain-manager-api
    cd domain-manager-api
    git remote add skeleton https://github.com/cisagov/skeleton-docker.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/workflows/build.yml README.md docker-compose.yml tests/container_test.py 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -11,6 +16,18 @@ updates:
...
+      - dependency-name: actions/checkout
+      - dependency-name: actions/setup-python
+      # Managed by cisagov/skeleton-docker
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/github-script
+      # - dependency-name: actions/upload-artifact
+      # - dependency-name: docker/build-push-action
+      # - dependency-name: docker/login-action
+      # - dependency-name: docker/setup-buildx-action
+      # - dependency-name: docker/setup-qemu-action
```
Comment (jsf9k) [inline .github/dependabot.yml]: These lines should be uncommented.

File: docker-compose.yml [hunk 2]
```diff
@@ -21,5 +21,6 @@ services:
 volumes:
   mongodb_data:
 
+
```
Comment (jsf9k) [inline docker-compose.yml]: Why was an extra blank line added here?

**Review by jsf9k — CHANGES_REQUESTED** [review]
I requested a couple of small changes.

File: .github/dependabot.yml (hunk 1 above)
Comment (itsmostafa) [reply .github/dependabot.yml]: @jsf9k i can go ahead and uncomment them, but fyi this change was inherited from the skeleton: https://github.com/cisagov/skeleton-docker/blob/develop/.github/dependabot.yml

File: docker-compose.yml (hunk 2 above)
Comment (itsmostafa) [reply docker-compose.yml]: looks like my linter prefers 2 lines. disabled it and resolved here: [6f4e2dc](https://github.com/cisagov/domain-manager-api/pull/486/commits/6f4e2dc309389576a2b5a900cab1b9f85dfb6f16)

File: .github/dependabot.yml (hunk 1 above)
Comment (itsmostafa) [reply .github/dependabot.yml]: @jsf9k i see that it is meant to be uncommented in downstream projects. resolved with [1d6a572](https://github.com/cisagov/domain-manager-api/pull/486/commits/1d6a572f6786d709cd55d22403ee864280f24805)

**Review by jsf9k — APPROVED** [review]

**Comment by jsf9k** [comment]
@dav3r and @mcdonnnj, can you take a look at this one and make sure I haven't missed anything?

**Review by mcdonnnj — APPROVED** [review]
As far as a Lineage pull request everything looks pretty in-order. I'm not sure what's going on with the `proxy_harness` Python package that sits in the root directory (and isn't part of the Docker image or the pytest configuration) as well as things like the `Domain Manager - Github.postman_collection.json` file.

**Comment by itsmostafa** [comment]
> As far as a Lineage pull request everything looks pretty in-order. I'm not sure what's going on with the `proxy_harness` Python package that sits in the root directory (and isn't part of the Docker image or the pytest configuration) as well as things like the `Domain Manager - Github.postman_collection.json` file.

@mcdonnnj we created the proxy harness to test web scraping and automation. it's not directly part of the application. the postman collection is just a json file for postman consisting of a collection of endpoints for testing the api. I may remove these in the future but not sure just yet. Am I good to merge?

**Pull request merged**